src/components/Drawer.js

```diff
@@ -164,8 +159,8 @@ const Drawer = React.createClass({
   },
 
   styles () {
-    return {
-      component: {
+    return StyleSheet.create({
```

Comment:
I agree that it would make testing easier but then we have to ensure argument order unless we plan to pass in an object but even then it requires the style function know about at least a key name.  I consider this at one point and decided that an internal private function knowing directly about props wasn't a bad thing.